apps/web/src/pages/integrations/components/ProviderCard.tsx

```diff
@@ -66,16 +66,17 @@ export function ProviderCard({
   );
 }
 
-const StyledButton = styled(Button)<{ theme: string }>`
-  background-image: ${({ theme }) =>
-    theme === 'dark'
+const StyledButton = styled(Button)<{ colorScheme: string }>`
+  background-image: ${({ colorScheme }) =>
+    colorScheme === 'dark'
```

Comment:
I would suggest you to leave `theme` as variable rather than `colorScheme`. The reason is that I had problems in the past when the property `colorScheme` is prop rendered in a React component, it complains it is not a valid HTML property. My guess is because of the casing because `colorscheme` takes it, but then it would be confusing for the developers and contributors.